site/content/integrate/apps/quick-start-js/_index.md

```diff
@@ -1,28 +1,26 @@
 ---
 title: "Quick start guide (JS)"
 heading: "Writing a Mattermost app in JavaScript"
-description: This quick start guide will walk you through the basics of writing a Mattermost app in JavaScript."
+description: "This quick start guide will walk you through the basics of writing a Mattermost app in JavaScript."
 weight: 5
 ---
 
 This quick start guide explains the basics of writing a Mattermost app. In this guide you will build an app that:
 
 - Contains a `manifest.json`, declares itself an HTTP application that acts as a bot, and attaches to locations in the user interface.
-- Attaches `send-modal` in its `bindings` to a button in the channel header, and `send` to a `/helloworld` command.
+- Attaches the form `send-modal` in its `bindings` to a button in the channel header, and the form `send` to a `/helloworld` command.
 - Contains a `send` function that sends a parameterized message back to the user.
 - Contains a `send-modal` function that forces displaying the `send` form as a modal.
 
 ## Prerequisites
 
-Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the enjoinment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc`. Please also ensure that `Bot Account Creation` and `OAuth 2.0 Service Provider` are enabled in the System Console.
+Before you can start with your app, you first need to set up a local developer environment following the [server](/contribute/server/developer-setup/) and [webapp](/contribute/webapp/developer-setup/) setup guides. You must enable the apps feature flag before starting the Mattermost server by setting the environment variable `MM_FEATUREFLAGS_AppsEnabled` to `true` by e.g. adding `export MM_FEATUREFLAGS_AppsEnabled=true` to your `.bashrc`. Please also ensure that `Bot Account Creation` and `OAuth 2.0 Service Provider` are enabled in the System Console.
```

Comment:
I think someone that doesn't want to pollute their `.bashrc` file may want to know the alternative of

`make run-server MM_FEATUREFLAGS_AppsEnabled=true`

Reply:
Is this something you meant to add? This thread is marked as resolved but I don't see the `make run-server` example in the PR

Reply:
Forgot to push. It's now in: f4c28c0b54a122d93c7169991c6de39137539692